Mathlib/ModelTheory/Semantics.lean

```diff
@@ -906,6 +906,59 @@ theorem realize_exs {φ : L.BoundedFormula α n} {v : α → M} :
       exact ⟨_, _, h⟩
 #align first_order.language.bounded_formula.realize_exs FirstOrder.Language.BoundedFormula.realize_exs
 
+@[simp]
+theorem _root_.FirstOrder.Language.Formula.realize_iAlls
+    [Finite γ] (f : α → β ⊕ γ)
+    (φ : L.Formula α) (v : β → M) : (φ.iAlls f).Realize v ↔
```

Comment:
```suggestion
    [Finite γ] {f : α → β ⊕ γ}
    {φ : L.Formula α} {v : β → M} : (φ.iAlls f).Realize v ↔
```
because it's an iff. What do you think?

Reply:
I don't feel strongly one way or the other. I feel like it will mainly be used with `simp` and `rw` anyway, so it doesn't matter and the arguments will rarely be given explicitly, but also, it will be rarely used with `Iff.mp` and `Iff.mpr`. I'll make the suggested change.